fixtures/dom/src/components/fixtures/text-inputs/README.md

```diff
@@ -0,0 +1,65 @@
+# Text Inputs
+
+There are a couple of important concepts to be aware of when working on text
+inputs in React.
+
+## `defaultValue` vs `value`
```

Comment:
perhaps worth noting that React explicitly supports changing the value of an input imperatively via JavaScript. This isn't an obvious invariant when making changing to input code, but required to probably support autofilling, password managers, and general DOM interop (unfortunately).